website/StandardAttributes.rst

```diff
@@ -0,0 +1,1388 @@
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> type </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Data types are defined by the type attribute. There are four
+         types:
+         <ul>
+           <li> Scan line images: indicated by a value of <tt>scanlineimage</tt> </li>
+           <li> Tiled images: indicated by a value of <tt>tiledimage</tt> </li>
+           <li> Deep scan line images:  indicated by a value of <tt>deepscanline</tt> </li>
+           <li> Deep tiled images:  indicated by a value of <tt>deeptile</tt> </li>      
+         </ul>
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> version </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> int </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Version 1 data for all part types is described in the section on the OpenEXR File Layout.
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> chunkCount </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> int </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+        Indicates the number of chunks in this part.  Required if the
+        multipart bit (12) is set.
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> tiles </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> TileDescription </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         This attribute is required only for tiled files. It specifies
+         the size of the tiles, and the file's level mode. 
+       </p>
+     </td>
+     <td style="vertical-align: top; width:100px"> required for multi-part images </td>
+   </tr>
+
+   </table>
+   </embed>
+
+Position and Orientation
+========================
+
+These attributes describe the position and orientation of the physical or CG
+camera at the time of capture. All are optional.
+
+.. raw:: html
+
+   <embed>
+   <table>
+   
+   <tr>
+     <td style="vertical-align: top; width:150px; font-weight:bold"> attribute name</td>
+     <td style="vertical-align: top; width:100px; font-weight:bold"> type </td>
+     <td style="vertical-align: top; width:500px; font-weight:bold"> definition </td>
+   </tr>
```

Comment:
I dropped the "status" column where everything is optional, because the stupid sphinx formatting limits with width of the table and the long definitions can use all the width they can get. I mention in the sentence above the tables that they're all optional. If it's confusing, it's easy to add the column back in. I just preferred to reclaim the extra width in place off an "optional" on every line. 

I'm totally open to redoing this entirely, but we definitely need something to distinguish required vs. optional attributes.

Reply:
I decided it would work better without the "status" column, and I added required underneath the attribute name where appropriate. How does that look?